runtime-modules/council/src/lib.rs

```diff
@@ -807,21 +837,24 @@ impl<T: Trait> EnsureChecks<T> {
         council_user_id: &T::CouncilUserId,
     ) -> Result<T::AccountId, Error<T>> {
         // ensure user's membership
-        let account_id = Self::ensure_user_membership(origin, council_user_id)?;
+        Self::ensure_user_membership(origin, council_user_id)?;
+
+        // ensure stake is unlockable
+        let staking_account_id = match Self::ensure_previous_stake_is_reusable(council_user_id)? {
+            Some(candidate) => candidate.staking_account_id,
+            None => return Err(Error::NoStake),
+        };
 
         // ensure user is not current council member
```

Comment:
If I get here, then there must have been a candidate for this member, hence this member cannot be serving on the current council, as there is only at most one candidate object per member at any time, and it is removed any time the member wins a seat in the council.

Reply:
You are right, I'll remove this condition.

Reply:
Done